### ISSUE TITLE
Missing transaction in my Electrum wallet

### ISSUE THREAD

**Issue by LePuppy** [spesmilo/electrum#3554]
Hello,
I've just created an Electrum wallet. I decided to tranfer first 0.002 btc from gdax. This first transaction was a success. Then I saw in order to deposit btc into gdax you need to transfer at least 0.01 btc. So I transfered 0.008 btc from gdax to my wallet (that's almost all I have). But I saw "unsuficient funds" an I thought the transfer was cancelled. Then I saw that I had almost no btc on gdax and that it confirmed the transfer in the history. Obviously, they are not in my wallet that still shows 2 mBTC from the first transaction. Any idea how it happend and more important : how to get my btc back ?
Thanks

PS : I'm using Google Authenticator so these transactions should have been secured
<img width="1079" alt="capture1" src="https://user-images.githubusercontent.com/20816887/34132392-e2d6856c-e44f-11e7-8f42-033b5fdc97cd.PNG">
<img width="421" alt="capture2" src="https://user-images.githubusercontent.com/20816887/34132394-e5462f78-e44f-11e7-946f-cada322fb712.PNG">

**Comment by SomberNight** [comment]
It's hard to follow the story here.
Have you sent two transactions from gdax to your electrum wallet, but only one shows up in the `History` tab in Electrum?
Do you have txids? Check them on a block explorer. If you find the txns, check the involved addresses.

**Comment by LePuppy** [comment]
I have sent two transactions to my wallet indeed. I don't know what txids is/are. Can you tell me what to do ? I can get the transactions ID on gdax.

**Comment by LePuppy** [comment]
You're right about the transactions. I've jsut check the first one on block explorer with the ID given on my wallet because the ID on gdax couldn't lead to any transaction. So I'm kinda blocked here since gdax is the only platform to say my second transaction worked out and I have no correct ID to check for it.

**Comment by LePuppy** [comment]
Nevermind it went on my coinbase account ! Thanks for helping !
